common/components/controllers/CorporateHackathonController.jsx

```diff
@@ -253,24 +253,27 @@ class CorporateHackathonController extends React.PureComponent<{||}, State> {
           </h3>
         </div>
 
-        {this._renderContactHackathon()}
+        
 
-        <div className="corporate-hackathon-saying corporate-bg-light col-12">
+        <div className="corporate-hackathon-saying col-12 corporate-bg-light">
```

Comment:
Did you mean to remove corporate-bg-light here or just reorder the class declarations?